src/test/java/io/confluent/connect/jdbc/source/integration/BaseConnectorIT.java

```diff
@@ -0,0 +1,228 @@
+
+    Map<String, String> props = new HashMap<>();
+    props.put("connector.class", JdbcSourceConnector.class.getName());
+    props.put("tasks.max", MAX_TASKS);
+    props.put("confluent.topic.replication.factor", "1");
+    props.put("confluent.topic.bootstrap.servers", connect.kafka().bootstrapServers());
+
+    return props;
+  }
+
+  protected Connection initializeJdbcConnection() throws SQLException {
+    return DriverManager.getConnection("jdbc:mysql://localhost:3307/db" , "user", "password");
+  }
+
+  protected void assertRecords(int end, ConsumerRecords<byte[], byte[]> records) {
+    int start = 0;
+    for (ConsumerRecord<byte[], byte[]> record : records) {
+      assertEquals(start, record.offset());
+      Struct valueStruct = ((Struct) (jsonConverter.toConnectData(KAFKA_TOPIC, record.value()).value()));
+      int id = valueStruct.getInt32("id");
+      assertEquals(start++, id);
+    }
+    if(start!=end){
+      throw new AssertionError("Invalid number of records found");
+    }
+  }
+
+  protected void addConnectionProperties(Map<String, String> props) {
+    props.put("connection.password", "password");
+    props.put("connection.user", "user");
+    props.put("connection.url", "jdbc:mysql://localhost:3307/db");
+    props.put("dialect.name", "MySqlDatabaseDialect");
+    props.put("value.converter", JsonConverter.class.getName());
+    props.put("topic.prefix",  "sql-");
+  }
+
+  protected void dropTableIfExists(String tableName) throws SQLException {
+    Statement st = connection.createStatement();
+    st.executeQuery("drop table if exists " + tableName);
+  }
+
+  protected void createTable() throws SQLException {
+    Statement st = connection.createStatement();
+    String sql = "CREATE TABLE " + KAFKA_TOPIC +
+            "(id INTEGER not NULL, " +
+            " first_name VARCHAR(255), " +
+            " last_name VARCHAR(255), " +
+            " age INTEGER, " +
+            " PRIMARY KEY ( id ))";
+    st.executeQuery(sql);
+  }
+
+  protected void createTimestampTable() throws SQLException {
+    Statement st = connection.createStatement();
+    String sql = "CREATE TABLE " + KAFKA_TOPIC +
+            "(id int not NULL, " +
+            " first_name VARCHAR(255), " +
+            " last_name VARCHAR(255), " +
+            " time1 TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP );";
+    st.executeQuery(sql);
+  }
+
+  protected void sendTestTimestampDataToMysql(int start, int numOfRecords) throws SQLException {
+    String sql = "INSERT INTO testtable(id,first_name,last_name) "
+            + "VALUES(?,?,?)";
+    PreparedStatement pstmt = connection.prepareStatement(sql);
+    for (int i=start; i<start+numOfRecords; i++) {
+      pstmt.setInt(1, i);
+      pstmt.setString(2, "FirstName");
+      pstmt.setString(3, "LastName");
+      pstmt.executeUpdate();
+    }
+    pstmt.close();
+  }
+
+  protected void sendTestDataToMysql(int start, int numOfRecords) throws SQLException {
```

Comment:
I thought we decided to send data only with timestamp and scrape this method off.